projects/components/src/table/table.component.ts

```diff
@@ -415,6 +415,12 @@ export class TableComponent
     if (changes.selections) {
       this.toggleRowSelections(this.selections);
     }
+
+    if (changes.loadingConfig) {
+      if (!this.loadingConfig?.load) {
+        this.loadingConfig = { ...this.loadingConfig, load: { loaderType: LoaderType.TableRow } };
```

Comment:
nice. Please add a todo here that we will improve this piece in the next iteration.